storybook/pages/button.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/594a9a3356.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../css/storybook.css">
+    <title>Button Component</title>
+</head>
+<body>
+    <span class="title">Navbar context</span>
+    <nav>
+        <button class="button">
+            Sample Btn
+            <i class="button__icon fa-solid fa-caret-down"></i>
+        </button>
+    </nav>
+    <span class="title">Form context</span>
+    <form>
+        <button class="button" type="submit">
```

Comment:
good intuition, bad memory

`type='submit'` refers to an input, not the button - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit

I know that it looks and behaves like a button, but it's an input.

Something like crocs and flip-flops :thinking: